README.md

```diff
@@ -291,7 +291,7 @@ func main() {
 	tempCreds.AuthorizedScopes = []string{
 		"queue:get-artifact:private/build/*",
 	}
-	queueClient, err := queue.New(tempCreds)
+	queueClient, err := tcqueue.New(tempCreds)
 	if err != nil {
 		// bug in code
 		log.Fatalf("SERIOUS BUG! Could not create client from generated temporary credentials: %v", err)
```

Comment:
Further down in this file there is a reference in code to "github.com/taskcluster/taskcluster-client-go/queue" which should be "github.com/taskcluster/taskcluster-client-go/tcqueue"

Reply:
I think that's fixed now